keyboards/ramonimbao/herringbone/pro/info.json

```diff
@@ -0,0 +1,321 @@
+
+                {"x": 15.5, "y": 0},
+
+                {"x": 0, "y": 1.25},
+                {"x": 1, "y": 1.25},
+                {"x": 2, "y": 1.25},
+                {"x": 3, "y": 1.25},
+                {"x": 4, "y": 1.25},
+                {"x": 5, "y": 1.25},
+                {"x": 6, "y": 1.25},
+                {"x": 7, "y": 1.25},
+                {"x": 8, "y": 1.25},
+                {"x": 9, "y": 1.25},
+                {"x": 10, "y": 1.25},
+                {"x": 11, "y": 1.25},
+                {"x": 12, "y": 1.25},
+                {"x": 13, "y": 1.25, "w": 2},
+
+                {"x": 15.5, "y": 1.25},
+
+                {"x": 0, "y": 2.25, "w": 1.5},
+                {"x": 1.5, "y": 2.25},
+                {"x": 2.5, "y": 2.25},
+                {"x": 3.5, "y": 2.25},
+                {"x": 4.5, "y": 2.25},
+                {"x": 5.5, "y": 2.25},
+                {"x": 6.5, "y": 2.25},
+                {"x": 7.5, "y": 2.25},
+                {"x": 8.5, "y": 2.25},
+                {"x": 9.5, "y": 2.25},
+                {"x": 10.5, "y": 2.25},
+                {"x": 11.5, "y": 2.25},
+                {"x": 12.5, "y": 2.25},
+                {"x": 13.75, "y": 2.25, "w": 1.25, "h": 2},
+
+                {"x": 15.5, "y": 2.25},
+
+                {"x": 0, "y": 3.25, "w": 1.75},
+                {"x": 1.75, "y": 3.25},
+                {"x": 2.75, "y": 3.25},
+                {"x": 3.75, "y": 3.25},
+                {"x": 4.75, "y": 3.25},
+                {"x": 5.75, "y": 3.25},
+                {"x": 6.75, "y": 3.25},
+                {"x": 7.75, "y": 3.25},
+                {"x": 8.75, "y": 3.25},
+                {"x": 9.75, "y": 3.25},
+                {"x": 10.75, "y": 3.25},
+                {"x": 11.75, "y": 3.25},
+                {"x": 12.75, "y": 3.25},
+                {"x": 15.5, "y": 3.25},
+
+                {"x": 0, "y": 4.25, "w": 1.25},
+                {"x": 1.25, "y": 4.25},
+                {"x": 2.25, "y": 4.25},
+                {"x": 3.25, "y": 4.25},
+                {"x": 4.25, "y": 4.25},
+                {"x": 5.25, "y": 4.25},
+                {"x": 6.25, "y": 4.25},
+                {"x": 7.25, "y": 4.25},
+                {"x": 8.25, "y": 4.25},
+                {"x": 9.25, "y": 4.25},
+                {"x": 10.25, "y": 4.25},
+                {"x": 11.25, "y": 4.25},
+                {"x": 12.25, "y": 4.25, "w": 1.75},
+
+                {"x": 14.25, "y": 4.5},
+
+                {"x": 15.5, "y": 4.25},
+
+                {"x": 0, "y": 5.25, "w": 1.25},
+                {"x": 1.25, "y": 5.25, "w": 1.25},
+                {"x": 2.5, "y": 5.25, "w": 1.25},
+                {"x": 3.75, "y": 5.25, "w": 6.25},
+                {"x": 10, "y": 5.25, "w": 1.5},
+                {"x": 11.5, "y": 5.25, "w": 1.5},
+
+                {"x": 13.25, "y": 5.5},
+                {"x": 14.25, "y": 5.5},
+                {"x": 15.25, "y": 5.5}
+            ]
+        },
+        "LAYOUT_all": {
```

Comment:
![image](https://user-images.githubusercontent.com/4365766/102920478-22356680-448b-11eb-8df0-f7eb8999aeff.png)
This is how `LAYOUT_all` renders, is this correct?

Reply:
There should be a tiny bit of space between the last three keys. I fixed that now, thanks.